src/indicators/drought/wrsi.py

```diff
@@ -0,0 +1,382 @@
+"""Code to download and process WRSI data
+
+The WRSI data is made publicly available through
+the USGS website. The data is dekadal and each
+dekadal dataset is shared through zip files.
+https://edcftp.cr.usgs.gov/project/fews/dekadal/
+
+These include two separate zones for West Africa:
+
+- WA: the largest region, covering from the coast at
+    5 degrees to around 15 degrees latitude, ranging
+    from the western coast of Senegal to the western
+    border of Sudan. This is specifically for croplands:
+    https://earlywarning.usgs.gov/fews/product/56
+- W1: smaller, ranges from around 11 degrees latitude
+    to around 17, and from the eastern border of Chad
+    to Senegal. This is specifically for rangelands:
+    https://earlywarning.usgs.gov/fews/product/57
+
```

Comment:
You mention it later but would also add here that data is only available from dekad 13 till 33 (add dates as well)